app/controllers/hearings_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class HearingsController < ApplicationController
+  def create
+    @hearing = HearingRecorder.call(params[:hearing][:id], hearing_params)
+
+    if @hearing
+      render json: @hearing, status: :created
```

Comment:
HMCTS expect a `location` header, however, this is omitted, since we do not have any provision for querying via the adaptor yet.
https://hmcts.github.io/restful-api-standards/#success-codes

Reply:
Really good catch! Going to add this as a ticket in the backlog so we don't forget about this bit.

Reply:
Also seems optional right? `You are free to return either an empty response or the created resource in conjunction with the Location header`

Reply:
It seemed a bit ambiguous, i.e , does it mean they need (both response + location) or (no response but _always_ location)
However, probably it is best to return an empty response, since otherwise we have to consider further questions about serialisation.

Reply:
Left the spike in the backlog anyways so we can be sure we will double check it with more time. (Even if it is discarded in 10 minutes :D )